src/docs/Foundations/Typography/BodyStyles/index.mdx

```diff
@@ -6,22 +6,25 @@ status:
   scss: 'wip'
 ---
 
-> The body patterns provide basic typographic styles for readable multilines text or paragraphs like a product description or an article.
+> A body style in typography is the main text part of a pattern or a page. It must be identity and provide basic typographic styles for readable multilines texts or paragraphs.
 
-Body styles are patterns meant to help you use consistent typographic hierarchy with the context and across responsive viewports. It's mainly reserved for users wishing to focus on tasks with added value and user experience. Text classes are families of several font-sizes and help create a hierarchy of contents.
+## Recommendation
 
-#### Body styles are defined with three factors :
+For readability and accessibility purposes we recommend you to use our regular value for body styles called **body medium**, corresponding to the following values:
```

Comment:
Maybe something simpler more like : 
"The default body style is the medium one."
We should also give usage examples for each body style size, maybe using schematics.

maybe using a structure like : 


## Body S

description 

preview

### examples of use : 

- do : example 1 with schematics
- don't : example 2 with schematics

## Body M

"the default body size" 

preview

### examples of use : 

- do : example 1 with schematics
- don't : example 2 with schematics

## Body L 

description 

preview

### examples of use : 

- do :example 1 with schematics
- don't : example 2 with schematics